src/error.rs

```diff
@@ -147,20 +147,20 @@ impl From<serde_json::Error> for Error {
     }
 }
 
-impl From<UrsaCryptoError> for Error {
-    fn from(err: UrsaCryptoError) -> Self {
+impl From<CryptoError> for Error {
+    fn from(err: CryptoError) -> Self {
         // let message = format!("Ursa Crypto Error: {}", Fail::iter_causes(&err).map(|e| e.to_string()).collect::<String>());
         let message = err.to_string();
         let kind = match err.kind() {
-            UrsaCryptoErrorKind::InvalidState => ErrorKind::InvalidState,
-            UrsaCryptoErrorKind::InvalidStructure | UrsaCryptoErrorKind::InvalidParam(_) => {
-                ErrorKind::Input
-            }
-            UrsaCryptoErrorKind::IOError => ErrorKind::IOError,
-            UrsaCryptoErrorKind::InvalidRevocationAccumulatorIndex => ErrorKind::InvalidUserRevocId,
-            UrsaCryptoErrorKind::RevocationAccumulatorIsFull => ErrorKind::RevocationRegistryFull,
-            UrsaCryptoErrorKind::ProofRejected => ErrorKind::ProofRejected,
-            UrsaCryptoErrorKind::CredentialRevoked => ErrorKind::CredentialRevoked,
+            CryptoErrorKind::InvalidState => ErrorKind::InvalidState,
+            // CryptoErrorKind::InvalidStructure | CryptoErrorKind::InvalidParam(_) => {
```

Comment:
@andrewwhitehead I've currently commented these out since they aren't defined in `anoncreds-clsignatures-rs`. I can issue a PR to add those `ErrorKind`s there if we want

Reply:
They aren't raised in `anoncreds-clsignatures`, so no need to worry about them :)

Reply:
Ok will remove them then